app/build.gradle

```diff
@@ -6,12 +6,14 @@ buildscript {
   dependencies {
     classpath 'com.neenbedankt.gradle.plugins:android-apt:1.7'
     classpath 'com.facebook.testing.screenshot:plugin:0.4.2'
```

Comment:
I think we don't need it no more :)

Reply:
I'd like to keep it here just in case during our trainings we face any issue related to shot.